dockerfiles/init/modules/codenvy/templates/codenvy.env.erb

```diff
@@ -50,7 +50,7 @@ JAVA_HTTPS_PROXY_PORT=-Dhttps.proxyPort=<%= @https_proxy_for_codenvy.gsub(/^http
 <% end -%>
 <% end -%>
 <% if ! @no_proxy_for_codenvy.empty? -%>
-JAVA_NO_PROXY=-Dhttp.nonProxyHosts='<%= @no_proxy_for_codenvy.gsub(/^https?\:\/\//, '').gsub(/^www./,'').gsub(',','|') %>|'
```

Comment:
Looks like this statement can be simplified: 
`.split(",").uniq.join("|") %>|'`